nepomuk/src/main/java/edu/asu/diging/gilesecosystem/nepomuk/core/service/handlers/AbstractFileHandler.java

```diff
@@ -104,5 +104,13 @@ public String getRelativePathInTypeFolder(IFile file) {
     
     protected abstract IFileStorageManager getStorageManager();
     
+    public void deleteFile(IFile file) {
+        try {
+          getStorageManager().deleteFile(file.getUsername(), file.getUploadId(), file.getDocumentId(), file.getFilename()); 
+        } catch (NepomukFileStorageException e) {
+          messageHandler.handleMessage("File could not be deleted.", e, MessageType.WARNING);
+        }
+        filesManager.deleteFile(file.getId());
```

Comment:
there needs to be a comment here why this method is called even if no file was deleted.

Reply:
comment needs to be broken over multiple lines (it's too long to easily read). Also, it doesn't explain why we delete it from the db even if the file deletion failed. Also, I think we need to distinguish between not deleted because there is no file (then we delete the db entry) and a file that exists but couldn't be delete for some reason (then we want to keep the db entry). In other words, I would not throw an exception if the file does not exists, and then only delete the db record when deletion was without exception.

Reply:
it's still too long (as you can see here, if GitHub has to wrap it, it's too long)